### PR TITLE
Refactor to get GitHub file

### PR DESCRIPTION
**Pull request by ncphillips**
The build fails because this depends on the changes from tinacms/tinacms#1329

### CODE REVIEW

**Comment by kendallstrautman** [comment]
Do you need a review with this @ncphillips?

**Comment by ncphillips** [comment]
Yeah, a review and some merges I guess

**Comment by ncphillips** [comment]
@kendallstrautman I just updated this. It's ready for review now.

File: pages/blog/[slug].tsx [hunk 1]
```diff
@@ -145,7 +145,7 @@ export const getStaticPaths: GetStaticPaths = async function() {
       const slug = fileToUrl(file, 'blog')
       return { params: { slug } }
     }),
-    fallback: true,
+    fallback: false,
```
Comment (ncphillips) [inline pages/blog/[slug].tsx:148]: @dwalkr this is what I was talking about

**Comment by ncphillips** [comment]
@dwalkr just switched this back to `fallback: true`

**Pull request merged**